src/SmartComponents/AddSystemModal/AddSystemModal.js

```diff
@@ -92,7 +117,13 @@ class AddSystemModal extends Component {
                             eventKey={ 1 }
                             title="Baselines"
                         >
-                            <BaselinesTable hasSelect={ true }/>
+                            <BaselinesTable
+                                hasMultiSelect={ true }
+                                tableType='CHECKBOX'
```

Comment:
What do you think about `tableId` rather than `type` since `type` tends to imply something that is from a defined set?

Reply:
so like: `tableId='CHECKBOX'`?